view/theme/frio/templates/comment_item.tpl

```diff
@@ -38,16 +38,10 @@
 					</button>
 				</li>
 				<li>
-					<button type="button" class="btn-link icon bb-url" style="cursor: pointer;" aria-label="{{$edurl}}" title="{{$edurl}}" onclick="insertFormatting('url',{{$id}});">
```

Comment:
As requested earlier, please keep the BBCode link button and instead replace the video button with an attach button with your new behavior and a paperclip icon.